app/src/main/res/layout/name_choices_dialog.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
```

Comment:
I don't think we need this LinearLayout. Let's try to remove it to improve performance.